build.gradle

```diff
@@ -30,8 +30,7 @@ allprojects {
         scalaVersion = '2.12.11'
         scalaTestVersion = '3.1.1'
         jacksonVersion = '2.12.0'
-        slf4jVersion = '1.7.7'
```

Comment:
Hmm, shoot I had missed your second post in the slack thread about this slf4j change. I did not make this change in passthrough, though I guess for whatever reason it was not needed there. Were you seeing a certain error that prompted the slf4j update? Not sure if I should be concerned about passthrough.

Reply:
Gotcha, I'll make a note to go make that change in passthrough on Monday so they match up.